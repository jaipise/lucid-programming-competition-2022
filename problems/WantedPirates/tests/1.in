6
Jamaica,Christopher Moody,Charlotte Lin-Lin,Sayyida Al Hurda,Monkey D. Luffy,Cool Runnings
France,Han Solo,Arthur Pendragon,Monkey D. Luffy,Jack Sparrow,Christopher Moody,Sayyida Al Hurda,Blackbeard
Dressrosa,Monkey D. Luffy,Edward Newgate,Christopher Moody,Jack Sparrow,Sayyida Al Hurda,Arthur Pendragon
Antigua,Long John Silvers,Blackbeard,Christopher Moody,Monkey D. Luffy,William Kidd,Sayyida Al Hurda,Wang Zhi,Stede Bonnet,Anne Bonny,Gol D. Roger
Saudi Arabia,Sayyida Al Hurda,Monkey D. Luffy,Lawrence of Arabia,Christopher Moody
Coruscant,Han Solo,Christopher Moody,Sayyida Al Hurda,Brownbeard,Barbosa,Monkey D. Luffy