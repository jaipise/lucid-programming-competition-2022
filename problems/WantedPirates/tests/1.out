Christopher Moody
Monkey D. Luffy
Sayyida Al Hurda
